src/exporter/DaemonMetricCollector.cc

```diff
@@ -381,7 +381,10 @@ void DaemonMetricCollector::dump_asok_metric(json_object perf_info,
     add_metric(builder, count, name + "_count", description + " Count", "counter",
                labels);
     json_value sum_value = perf_values.as_object()["sum"];
-    add_double_or_int_metric(builder, sum_value, name + "_sum", description + " Total",
+    add_double_or_int_metric(builder, sum_value, name + "_sum", description + " total",
+                             metric_type, labels);
+    json_value avg_value = perf_values.as_object()["avgtime"];
+    add_double_or_int_metric(builder, avg_value, name + "_avg", description + " average",
```

Comment:
@cbodley @idryomov This change reflects that were weren't sending the `avgtime` metric to prometheus from our time based counters that were being dumped via `counter dump`.  

@avanthakkar's rationale for not including these was that the average could be computed from the total time, and the total number of entries also dump in a time based counters.

As tech leads do you have any strong opinion on including or excluding these from what the exporter is sending to Prometheus? I'm inclined to remove it just to send out less data, but I don't feel strongly either way.

Reply:
because the average can be derived from values we're already sending to prometheus (`sum / avgcount`) it shouldn't be necessary to send this too

p.s. note that "avgtime" is specific to `PERFCOUNTER_TIME`, and doesn't apply for `PERFCOUNTER_U64`

Reply:
Agree with Casey -- I don't think anything needs changing here.